app/views/partials/_admin_login.html.erb

```diff
@@ -5,3 +7,5 @@
     <%= f.submit "Login" %>
   <% end %>
 </div>
+</div>
+</div>
```

Comment:
Are the divs actually misaligned here? Or is this just a weird github view of the code?